src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -673,19 +677,18 @@ public void SetThreadPrincipal()
         private void CopyTestAssemblies()
         {
             string destTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "AssemblyResolveTests", "AssemblyResolveTests.dll");
-            if (File.Exists("AssemblyResolveTests.dll"))
+            Console.WriteLine($"destinationPat={destTestAssemblyPath}");
+            if (!File.Exists(destTestAssemblyPath) && File.Exists("AssemblyResolveTests.dll"))
             {
                 Directory.CreateDirectory(Path.GetDirectoryName(destTestAssemblyPath));
-                File.Copy("AssemblyResolveTests.dll", destTestAssemblyPath, true);
-                File.Delete("AssemblyResolveTests.dll");
+                File.Copy("AssemblyResolveTests.dll", destTestAssemblyPath, false);
```

Comment:
I'm not sure why did we remove the file and forced overwrite - this works when there is one directory per tests but that's not what's happening on Desktop